docs/tutorials/intro.md

```diff
@@ -67,7 +67,7 @@ fn main() {
     let loc: Locale = "ES-AR".parse()
         .expect("Failed to parse locale.");
 
-    if loc.id.language == "es" {
+    if loc.id.language.as_str() == "es" {
```

Comment:
We usually do the comparison like `loc.id.language == language!("es")`

We should also use the macro for the locale

Reply:
Ah we haven't introduced that yet